src/components/UserInfo/UserInfo.scss

```diff
@@ -1 +1,3 @@
-//
+.UserInfo {
```

Comment:
Consider using nested selectors to better organize your SCSS code. For example, if there are multiple elements within the UserInfo component that need styling, nesting can help make the code more readable and maintainable. 📚

Reply:
????? There is only one selector.